tests/e2e/unified_test.go

```diff
@@ -475,11 +475,20 @@ func runScenario(ctx context.Context, t *testing.T, testPause bool, fixture reso
 
 					// Replace any dynamic IDs that appear in URLs
 					for _, event := range events {
-						url := event.Request.URL
+						u := event.Request.URL
 						for k, v := range pathIDs {
-							url = strings.ReplaceAll(url, "/"+k, "/"+v)
+							u = strings.ReplaceAll(u, "/"+k, "/"+v)
+						}
+
+						// Specific to Compute
+						// Terraform uses the /beta/ endpoints, but mocks and direct controller should use /v1/
+						// This special handling to avoid diffs in http logs.
+						// This can be removed once all Compute resources are migrated to direct controller.
+						basePath := "https://compute.googleapis.com/compute"
```

Comment:
Oh I see!  I guess this is a nice trick also.  @yuwenma  what do you think?

Reply:
works for me.